File: UIProfile/app/src/main/java/com/example/uiprofile/MainActivity.java
package com.example.uiprofile;

import android.os.Bundle;
import android.widget.ImageButton;
import android.widget.Toast;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        // Back
        ImageButton btnBack = findViewById(R.id.btnBack);
        btnBack.setOnClickListener(v -> {
            // Quay lại màn
            finish();
        });

        /* Edit */
        ImageButton btnEdit = findViewById(R.id.btnEdit);
        btnEdit.setOnClickListener(v -> Toast.makeText(this, "Edit", Toast.LENGTH_SHORT).show());
    }
}

File: Week2/app/src/main/java/com/example/mobileapplicationexcercise/MainActivity.kt
package com.example.mobileapplicationexcercise

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnBai1 = findViewById<Button>(R.id.btnBai1)
        val btnBai2 = findViewById<Button>(R.id.btnBai2)
        val btnBai3 = findViewById<Button>(R.id.btnBai3)

        btnBai1.setOnClickListener {
            val intent = Intent(this, Bai1Activity::class.java)
            startActivity(intent)
        }

        btnBai2.setOnClickListener {
            val intent = Intent(this, Bai2Activity::class.java)
            startActivity(intent)
        }

        btnBai3.setOnClickListener {
            val intent = Intent(this, Bai3Activity::class.java)
            startActivity(intent)
        }
    }
}
